src/restconfig/src/main/java/org/geoserver/service/rest/ServiceSettingsResource.java

```diff
@@ -76,18 +66,19 @@ protected Object handleObjectGet() throws Exception {
                     Status.CLIENT_ERROR_NOT_FOUND);
         }
 
-        return service;
+        return service.clone(true);
```

Comment:
Given this is configuration, wouldn't it make more sense to show the parameters in the output, accept them in input, and then fix validation like it's done in the UI?